tests/Fixtures/Elasticsearch/Model/Book.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Tests\Fixtures\Elasticsearch\Model;
+
+use ApiPlatform\Elasticsearch\Filter\MatchFilter;
+use ApiPlatform\Elasticsearch\Filter\OrderFilter;
+use ApiPlatform\Elasticsearch\Metadata\ElasticsearchDocument;
+use ApiPlatform\Metadata\ApiFilter;
+use ApiPlatform\Metadata\ApiProperty;
+use ApiPlatform\Metadata\ApiResource;
+use Symfony\Component\Serializer\Annotation\Groups;
+
+#[ApiResource(normalizationContext: ['groups' => ['book:read']], persistenceMeans: new ElasticsearchDocument('book'))]
```

Comment:
I think that we need to discuss naming on these. We suggested `stateOptions` as it is broader. Why the `persistenceMeans` choice? 

About `ElasticsearchDocument` I'd suggest ElasticsearchStateOptions instead.